src/style.css

```diff
@@ -0,0 +1,100 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("/fonts/Avenir-Book.ttf") format("truetype"),
+    url("/fonts/Avenir-Heavy.ttf") format("truetype");
+}
+
+body {
+  font-weight: 300;
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.search-input-small,
+.search-input,
+.search-form {
+  border: none;
+  margin: 0;
+  padding: 0;
+  background-color: transparent;
+}
+
+.search-input {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+  display: inline-block;
+  margin-left: 62px;
+  font-size: 16px;
+  color: #3d4e61;
+  font-weight: 500;
+}
+
+.search-input-small {
+  margin-left: 33px;
+  font-size: 14px;
+  font-style: normal;
+}
+
+.search-input-small:focus,
+.search-input:focus {
+  outline: none;
+  background-color: transparent;
+}
+
+.search-input-small::placeholder,
+.search-input::placeholder {
+  color: #3d4e61;
+  font-weight: 300;
+  font-style: normal;
+}
+
+.search-form {
+  margin-top: 20px;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-big {
+  height: 70px;
+  line-height: 69px;
+}
+
+.search-small {
+  height: 42px;
+  line-height: 41px;
+}
```

Comment:
try do not hardcode height
your box should be stretched by inner elements

Reply:
can i hardcode inner elements?